src/functionalTest/resources/responses/response-caseworker-confirm-payment-bailiff.json

```diff
@@ -0,0 +1,26 @@
+{
+  "data":{
+    "divorceOrDissolution":"divorce",
+    "applicant1FirstName":"John",
+    "applicant1LastName":"Smith",
+    "applicant1Email":"simulate-delivered@notifications.service.gov.uk",
+    "applicant2Gender":"female",
+    "solServiceTruthStatement":"I believe that the facts stated in the application are true.",
+    "solPaymentHowToPay":"feePayByAccount",
+    "alternativeServiceType":"bailiff",
+    "servicePaymentFeeOrderSummary": {
+      "PaymentReference":null,
+      "PaymentTotal":"27500",
```

Comment:
I think we need to set this in pounds. @linusnorton I remember you had to change the amount to pounds for PBA do we need to do same here

Reply:
The CreditAccountPaymentRequest.setamount method is still dividing the ordersummary payment total by 100